### PR TITLE
dklatestcode

### PR DESCRIPTION
**Pull request by danishkhan117**
i have just changed my code pls

### CODE REVIEW

File: src/main/resources/config/config.properties [hunk 1]
```diff
@@ -1,4 +1,4 @@
-browser = firefox
```
Comment (shahadat459) [inline src/main/resources/config/config.properties:1]: please dont change browser

Comment (shahadat459) [inline src/main/resources/config/config.properties:1]: please dont change browser

**Review by shahadat459 — COMMENTED** [review]
do the req changes

**Review by shahadat459 — COMMENTED** [review]
do the req changes

**Comment by danishkhan117** [comment]
ok ill check

**Pull request merged**